Simulator/sensormodels.py

```diff
@@ -120,16 +120,16 @@ def whitenoise2(mean = 0, cov = 0, dims = None):
     return np.random.multivariate_normal(mean*np.ones(dims), cov)
     
 
-def getTmatrix(scaleF, caSense):
+def getTmatrix(scalefactor, crossaxis_sensitivity):
     """
     Inputs:
-        scaleF: scale factor for a particular sensor
-        caSense: cross-axis sensitivity for particular sensor
+        scalefactor: scale factor for a particular sensor
+        crossaxis_sensitivity: cross-axis sensitivity for particular sensor
```

Comment:
I'm not sure it's necessary to have such a long variable name when there is a description here, or whether it's necessary to have a description that's just a restatement of the variable name. Ultimately, it doesn't matter though. I think Tomer put in that original short variable name to keep things cleaner.

Reply:
I personally have no preference. The change was inspired by this discussion: [link](https://github.com/spacecraft-design-lab-2019/groundtruth-simulator/issues/58).

Let me know if you both would like to change it back.

Reply:
gotcha, good with me then. Perhaps we should put a note on what cross axis sensitivity and scale factor are, or a link to the datasheet (https://ae-bst.resource.bosch.com/media/_tech/media/datasheets/BST-BMX160-DS000.pdf), but that is up to you.

Reply:
Good idea to include a link; I'll throw it in there. The datasheet does have cross axis sensitivity in there (searched for it) but doesn't use the phrase scale factor. Do you recall what it calls it?